packages/static/kairos-overlay-files/files/system/oem/00_home_dir_owner_fix.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Fix home directory permissions (kairos issue #2797)"
+stages:
+  initramfs.after:
+    - name: "Fix permissions"
+      commands:
+        - |
+          # Iterate over users in /etc/passwd and chown their directories
+          awk -F: '$3 >= 1000 && $6 ~ /^\/home\// {print $1, $6}' /etc/passwd | while read -r user homedir; do
+              if [ -d "$homedir" ]; then  # Check if the home directory exists
+                  echo "Changing ownership of $homedir to $user"
+                  chown -R "$user":"$user" "$homedir"
```

Comment:
`uid $user -i` to get the user id, same for the group? so we set the user:group to numbers instead of names?

Reply:
Does it make any difference? I think it doesn't matter how you refer to the user and group, in the end it's going to set the ownership the same way. E.g.

```
~ $ sudo chown 1000:1001 deleteme
~ $ ls -liah deleteme
9717519 -rw-r--r-- 1 dimitris dimitris 0 Αυγ  14 14:32 deleteme
~ $ sudo chown dimitris:dimitris deleteme
~ $ ls -liah deleteme
9717519 -rw-r--r-- 1 dimitris dimitris 0 Αυγ  14 14:32 deleteme
```